phoenix-core/src/main/java/org/apache/phoenix/util/ScanUtil.java

```diff
@@ -957,4 +962,35 @@ public static int getClientVersion(Scan scan) {
     public static void setClientVersion(Scan scan, int version) {
         scan.setAttribute(BaseScannerRegionObserver.CLIENT_VERSION, Bytes.toBytes(version));
     }
+
+    public static void getDummyResult(byte[] rowKey, List<Cell> result) {
+        result.clear();
+        KeyValue keyValue =
+                KeyValueUtil.newKeyValue(rowKey, 0,
+                        rowKey.length, EMPTY_BYTE_ARRAY, EMPTY_BYTE_ARRAY,
+                        0, EMPTY_BYTE_ARRAY, 0, EMPTY_BYTE_ARRAY.length);
+        result.add(keyValue);
```

Comment:
What if the actual data contains this exact keyValue? Is that possible?

Reply:
Is it possible to have a column with a column family name and a column qualifier name such that the byte array representation of these names map to empty byte array? I assume it is not possible. How do you even express that within a SQL statement?

Reply:
I don't think so, just asking to make sure.